### PR TITLE
Add radical flux tolerance

### PR DESCRIPTION
**Pull request by mjohnson541**
This PR adds the radical flux tolerance to RMG. It is dependent on: https://github.com/ReactionMechanismGenerator/ReactionMechanismSimulator.jl/pull/200.

### CODE REVIEW

**Comment by alongd** [comment]
@mjohnson541, could you also document the new tolerance?

**Comment by mjohnson541** [comment]
Will do! Resurrecting old commits right now, mostly threw this up to run tests.

File: documentation/source/users/rmg/input.rst [hunk 1]
```diff
@@ -450,7 +450,62 @@ or :ref:`pruning <pruning>` can be turned on to speed up your simulation at a sl
...
+Branching criterion identifies important edge reactions based on the impact they are expected to
+have on the concentrations of core species. The branching specific tolerances shown in the example below
+are the recommended tolerances. These tolerances are much less system dependent than the flux tolerance and
+the recommended values are usually sufficient.
+
+For example ::
+
+	model(
+		toleranceMoveToCore=0.1,
+		toleranceBranchReactionToCore=0.001,
```
Comment (hwpang) [inline documentation/source/users/rmg/input.rst:469]: Could you fix the indentation here?

**Review by hwpang — CHANGES_REQUESTED** [review]
Thanks for the PR! I only have one minor comment.

File: documentation/source/users/rmg/input.rst (hunk 1 above)
Comment (mjohnson541) [reply documentation/source/users/rmg/input.rst]: I've made all the indentation consistent with the original pruning indentation. It looks significantly different in the text editor than it does on github.

**Review by hwpang — APPROVED** [review]

**Pull request merged**